docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,539 @@
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    app_control_destroy(app_control);
+    ```
+
+- Main type:
+
+  You can specify a main item among several items. If there are multiple text items and a main item is specified, then you can display the items in particular.
+  The main type contains only title, contents, icon, and button.
+    ```
+    noti_ex_item_h text_item = NULL;
+    noti_ex_item_h text_item2 = NULL;
+    noti_ex_item_h group_item = NULL;
+
+    noti_ex_item_group_create(&group_item, "group_id");
+    noti_ex_item_text_create(&text_item, "text_id", "title", "hyperlink");
+    noti_ex_item_text_create(&text_item2, "text_id2", "contents", "hyperlink");
+    noti_ex_item_group_add_child(group_item, text_item);
+    noti_ex_item_group_add_child(group_item, text_item2);
+
+    ret = noti_ex_item_set_main_type(group_item, "text_id", type);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="expanded"></a>
+## Creating Expanded Notification
+
+You can provide a more complex form of notification. You can also specify the display order of each item.
+The display order of each item depends on the order in which the child is added to the group:
+
+    ```
+    noti_ex_item_h group_item = NULL;
+    noti_ex_item_h image_item_icon = NULL;
+    noti_ex_item_h image_item1 = NULL;
+    noti_ex_item_h image_item2 = NULL;
+    noti_ex_item_h title_item = NULL;
+    noti_ex_item_h contents_item = NULL;
+    noti_ex_item_h button_item = NULL;
+    noti_ex_item_h selector_item = NULL;
+
+    ret = noti_ex_item_image_create(&image_item_icon, "icon_id", icon_path);
+    ret = noti_ex_item_image_create(&image_item1, "image_id1", image1_path);
+    ret = noti_ex_item_image_create(&image_item2, "image_id2", image2_path);
+    ret = noti_ex_item_text_create(&title_item, "title_id", "Jone", NULL);
+    ret = noti_ex_item_text_create(&contents_item, "contents_id", "How are you ...", NULL);
+    ret = noti_ex_item_button_create(&button_item, "button_id", "button_title");
+    ret = noti_ex_item_input_selector_create(&selector_item, "selector_id");
+    const char *contents[] = {"What's up?", "Nice",. "On my way"};
+    ret = noti_ex_item_input_selector_set_contents(selector_item, contents, 3);
+
+    ret = noti_ex_item_group_create(&group_item, "group_id");
+    ret = noti_ex_item_group_add_child(group_item, image_item_icon);
+    ret = noti_ex_item_group_add_child(group_item, title_item);
+    ret = noti_ex_item_group_add_child(group_item, contents_item);
+    ret = noti_ex_item_group_add_child(group_item, image_item1);
+    ret = noti_ex_item_group_add_child(group_item, image_item2);
+    ret = noti_ex_item_group_add_child(group_item, button_item);
+    ret = noti_ex_item_group_add_child(group_item, selector);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+**Figure: Expanded Notification**
+
+![Expanded Notification](./media/expanded_notification.png)
```

Comment:
Just question. Don't we need time item also under title item according to the sample code?

Reply:
Time items are automatically saved unless you add them.